pages/linux/mesg.md

```diff
@@ -0,0 +1,15 @@
+# mesg
+
+> Check or set a terminal's ability to receive messages from other users, usually from the write command.
+
```

Comment:
Maybe we could add an additional like below the description line like this?

> See also `write`.

Reply:
Would a reciprocal link from write to mesg be appropriate?

Reply:
Good idea! Looks like you forget the trailing full stop though

Reply:
@sbrl I've spent 15 minutes wringing my hands about this: should a change to [write.md](https://github.com/tldr-pages/tldr/blob/master/pages/linux/write.md) be in a separate PR? I realize this is a silly concern, but I'm keen to get it right.

Reply:
@lbonanomi, I would create it as a separate PR. :+1:

Reply:
Not a silly question, @lbonanomi! Generally, edits to each command should each have their own PRs. The exception here is if you're making the same change to many pages, such as in #2896 for example.